ui/src/me/components/ProfileOverlay.tsx

```diff
@@ -0,0 +1,84 @@
+// Libraries
+import React, {PureComponent, ChangeEvent} from 'react'
+import {withRouter, WithRouterProps} from 'react-router'
+import {connect} from 'react-redux'
+
+// Components
+import {Form, Input, Button, ComponentColor} from '@influxdata/clockface'
+import {Overlay} from 'src/clockface'
+
+// Types
+import {AppState} from 'src/types'
+import {ComponentSize, ComponentStatus} from '@influxdata/clockface'
+
+interface StateProps {
+  me: AppState['me']
```

Comment:
This is a good example of where we should use a small interface that represents what we need for the component rather than pulling a type from AppState.